hack/lib/test.sh

```diff
@@ -33,8 +33,9 @@ kube::test::get_object_assert() {
   local request=$2
   local expected=$3
   local args=${4:-}
```

Comment:
I will checkout everywhere where to function is called and see if it can be done.